### PR TITLE
chore: renewal authz api

### PR DESCRIPTION
**Pull request by hjianbo**
see: https://github.com/emqx/emqx/pull/8381

### CODE REVIEW

File: en_US/security/authz/http.md [hunk 1]
```diff
@@ -7,13 +7,24 @@ HTTP authorizer delegates authorization to a custom HTTP API.
...
 For untrusted environments, HTTPS should be used.
 :::
 
+## Required HTTP Response Format
+
+- Response `Content-Type` can be `application/x-www-form-urlencoded` or `application/json`.
+- If the HTTP Status Code is `200`, the authorization result is granted by HTTP Body. It depends on the value of the `result` field:
+    - `allow`: Allow Publish or Subscribe.
+    - `deny`: Deny Publish or Subscribe.
+    - `ignore`: Ignore this request, It will be handed over to the next authorizer.
```
Comment (thalesmg) [inline en_US/security/authz/http.md]:
```suggestion
    - `ignore`: Ignore this request, it will be handed over to the next authorizer.
```

**Review by thalesmg — DISMISSED** [review]

File: en_US/deploy/upgrade.md [hunk 2]
```diff
@@ -192,7 +192,7 @@ Users can use the data import API to import data from older versions into EMQX 5
 **Success response status code:**
 
 ```shell
-200 or 204
+200
```
Comment (tigercl) [inline en_US/deploy/upgrade.md]:
```suggestion
200 or 204
```

File: zh_CN/deploy/upgrade.md [hunk 3]
```diff
@@ -179,7 +179,7 @@ authentication = [
 **成功响应状态码:**
 
 ```shell
-200 或 204
+200
```
Comment (tigercl) [inline zh_CN/deploy/upgrade.md]: 200 或 204

Comment (hjianbo) [inline zh_CN/deploy/upgrade.md]:
```suggestion
200 或 204
```

**Review by tigercl — APPROVED** [review]

**Pull request merged**